python/rtypes/dataframe/object_manager.py

```diff
@@ -286,7 +305,7 @@ def extend(self, tp_obj, objs):
         return records
 
     def get_one(self, tp_obj, oid, parameter):
-        obj_map = self.__get(tp_obj, parameter)
+        obj_map = self.__get(tp_obj, parameter)adjust
```

Comment:
Something wrong with the syntax here. Fix. Also run existing test cases to make sure you are not breaking anything,